pulp_docker/tests/functional/api/test_crud_publishers.py

```diff
@@ -34,7 +34,7 @@ def tearDownClass(cls):
 
     def test_01_create_publisher(self):
         """Create a publisher."""
-        body = gen_docker_publisher()
```

Comment:
There is no need to have a specialized gen function, `gen_publisher` works for docker